static/app/views/explore/charts/confidenceFooter.tsx

```diff
@@ -9,17 +9,23 @@ import {defined} from 'sentry/utils';
 type Props = {
   confidence?: Confidence;
   sampleCount?: number;
+  topEvents?: number;
 };
 
-export function ConfidenceFooter({sampleCount, confidence}: Props) {
+export function ConfidenceFooter({sampleCount, confidence, topEvents}: Props) {
+  const prefix =
+    defined(topEvents) && topEvents > 0
+      ? t('Chart for top %s groups', topEvents)
+      : t('Chart');
   return (
     <Container>
       {!defined(sampleCount)
-        ? t('* Chart extrapolated from \u2026')
+        ? tct('* [prefix] extrapolated from \u2026', {prefix})
```

Comment:
Technically we shouldn't be translating and assuming parts of the sentence can be injected in this way, right? Each combination of options needs to be enumerated